### PR TITLE
[Bugfix] Improve CompGCN

### PR DESCRIPTION
**Pull request by nxznm**
## Description

I make some improvements in the DGL implementation of CompGCN, including:

1. larger batch size for faster training:

```
parser.add_argument('--batch', dest='batch_size', default=1024, type=int, help='Batch size')
```

2. improve the efficiency of GPU:

```
train_loss.append(tr_loss.item())
```

3. larger `kill_cnt` for early stop: [Because I find that `kill_cnt=25` is not enough on WN18RR, CompGCN need more iterations to converge on WN18RR.]

```
kill_cnt += 1
if kill_cnt > 100:
    print('early stop.')
    break
```

4. support pytorch version >= 1.8.0:

```
return th.fft.irfftn(th.conj(th.fft.rfftn(a, (-1))) * th.fft.rfftn(b, (-1)), (-1))
```

I also update the README.MD, here are my results:

* Dependencies

  ````
  pytorch 1.9.0
  dgl 0.7.1
  numpy 1.20.3
  ordered_set 4.0.2
  ````

* Link Prediction Results

  | Dataset |    FB15k-237     |     WN18RR     |
  |---------| ------------------------ | ------------------------ |
  | Metric |  Paper  / ours (dgl) |  Paper  / ours (dgl) |
  |  MRR  |  0.355  /  0.348  |  0.479  /  0.466  |
  |  MR  |   197  /   208   |  3533  /   3542  |
  | Hit@10 |  0.535  /  0.527   |  0.546  /  0.525  |
  | Hit@3 |  0.390  /  0.380  |  0.494  /  0.476  |
  | Hit@1 |  0.264  /  0.259  |  0.443  /  0.435  |
## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [x] To the best of my knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Review by mufeili — APPROVED** [review]
LGTM. Thanks.
